docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper:latest
```

Comment:
duplicate the images in ci/docker/zookeeper/ e.g. copy the contents of wurstmeister/zookeeper/Dockerfile into ci/docker/zookeeper/Dockerfile so we have a local copy